lib/src/lexer/token.dart

```diff
@@ -131,7 +131,19 @@ enum TokenType {
   unitLiteral,
 
   /// The verum token (`⊤`).
-  verum;
+  verum,
+
+  /// Integer literals:
+  /// 0100
+  /// 1094812
+  /// 100_000
```

Comment:
```suggestion
  /// The integer literal.
  /// 
  /// It follows the following grammar:
  /// ```ebnf
  /// <digit_separator> ::= "_"
  /// <integer_literal> ::= <digit>+ ( <digit_separator> <digit>+ )*
  /// ```
  ///
  /// Some examples of valid integer literals:
  /// ```
  /// 0100
  /// 1094812
  /// 100_000
  /// ```
```

Reply:
Do the same with `doubleLiteral`.